demo/demo.py

```diff
@@ -137,11 +180,16 @@
 
     if potential_rect.collidelist(solid_blocks_on_players_index) != -1:
         print("colliding!")
+        x_speed = y_speed = 0
     else:
         y_coord = potential_y_coord
         x_coord = potential_x_coord
         camera.scroll_to(potential_rect)
         #camera.scroll_absolute(x_coord - 10, y_coord - 10)
+
+    source.x = x_coord + animated_sprite.image.get_size()[0] / 2
```

Comment:
`x_coord` will not work with demo once merged to `nextrelease`. instead, it'll have to be `animated_collisionsprite.rect.left` or whatever